operator/pkg/controllers/psstrategy/worker.go

```diff
@@ -39,11 +38,12 @@ func newWorkerManager() *WorkerManager {
 }
 
 func (m *WorkerManager) generateWorker(job *elasticv1alpha1.ElasticJob, replicaIndex int32) *corev1.Pod {
-	workerSpec := job.Spec.ReplicaSpecs[ReplicaTypeWorker]
-	workerTemplate := &workerSpec.Template
-	workerName := m.generatePodName(job, replicaIndex)
-	pod := m.GeneratePod(job, workerTemplate, workerName)
-	pod.Labels[LabelRestartCount] = fmt.Sprintf("%d", workerSpec.RestartCount)
+	spec := job.Spec.ReplicaSpecs[ReplicaTypeWorker]
```

Comment:
check if "ReplicaSpecs == nil" ?

Reply:
No, `ReplicaSpecs` has been defined as a map and it will be initialized. But, we need to check whether  ReplicaTypeWorker is in it.